### DISCUSSION TITLE
Parallel calls with Python

### DISCUSSION THREAD

**Discussion by Kee2u** [microsoft/semantic-kernel#5751] [Q&A] [answered]
Hello all,
Is there a way to parallelize calls to the LLM in SK python? I saw the assistants video here: https://devblogs.microsoft.com/semantic-kernel/openai-assistants-the-power-of-templated-assistant-instructions/ that showed how to do it in C# so was wondering if there was a way to do it with python. I believe assistants are not ready yet in python.

**Comment by sophialagerkranspandey** [comment, marked as answer]
Assistants aren't available in python yet but in the meantime, you can use parallel function calling with automatic function calling.
